### PR TITLE
[Feature request] Added Sponsor logo to footer

### PR DESCRIPTION
**Pull request by anmode**
# Descripción

Added sponsor logo to footer amazon,f5,replit

Fixes: feature request

# Checklist:

- [x] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [x] Se corrieron todas las pruebas y pasaron.
- [x] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [ ] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

File: frontend/www/js/omegaup/components/common/Footer.vue [hunk 1]
```diff
@@ -26,6 +26,24 @@
                 />
               </a>
             </li>
+            <li class="mt-1">
+              <a href="https://www.f5.com/" target="_blank">
+                <img
+                  class="sponsor-logo"
+                  src="/media/homepage/f5_logo.png"
```
Comment (lhchavez) [inline frontend/www/js/omegaup/components/common/Footer.vue:33]: can the file be renamed to `f5_logo.png`? even though Windows doesn't care about uppercase/lowercase in file names, macOS and Linux do. Our server runs Linux, which means that the logo will no longer appear for anybody!

Comment (lhchavez) [reply frontend/www/js/omegaup/components/common/Footer.vue:33]: also, can the aspect ratio of the image be kept as a square? (resize it to 400x400)

Comment (anmode) [reply frontend/www/js/omegaup/components/common/Footer.vue:33]: Ok sure! I'll rename it to f5_logo.png and change it's ratio to 400×400
Actually this is png so white background should not come. I have downloaded img as png with no background but aften running it on localhost white appearing what should i do now??

**Review by lhchavez — COMMENTED** [review]
now it's way larger D: it's 3000x2000 now, should be 400x400.

**Comment by anmode** [comment]
> now it's way larger D: it's 3000x2000 now, should be 400x400.

I m getting an error in seconds commit.
Trying to solve it. I have posted it on discord sever.

**Comment by lhchavez** [comment]
> Would this Pr can be merged so i can clear my last phase?
> 314×314 looks great.
> In that time I'll set my environment again.
> It's just a request.

you can create a new branch in the meantime!

**Comment by lhchavez** [comment]
> > Would this Pr can be merged so i can clear my last phase?
> > 314×314 looks great.
> > In that time I'll set my environment again.
> > It's just a request.
> 
> you can create a new branch in the meantime!

just to be clear, we're still waiting on the image to be resized and cropped so that it's a smaller square. can't merge before that!

**Comment by anmode** [comment]
> > > Would this Pr can be merged so i can clear my last phase?
> > > 314×314 looks great.
> > > In that time I'll set my environment again.
> > > It's just a request.
> > 
> > 
> > you can create a new branch in the meantime!
> 
> just to be clear, we're still waiting on the image to be resized and cropped so that it's a smaller square. can't merge before that!

Yes i m pushing it soon. Sorry for delay!

**Pull request closed without merging**